### PR TITLE
Cat menu updates

### PR DESCRIPTION
**Pull request by axelsson**
I updated the menu with the new layout to match CAT. If the user has access to anything in the "business section" the menu will be split between their personal links and the business links.

I removed the current business page and all related pages.
The thing that's still left from the previous work with the business page is the two rake tasks (`lib/tasks/add_climate_report_invoice_email_as_organization_admin.rake` `lib/tasks/connect_climate_reports_to_organizations.rake`) and two emails (`spec/mailers/previews/business_mailer_preview.rb`) since we might be needing something like that with the new user flow.

The link to CAT is still limited to `current_user.organization&.key == 'goclimate'` so this should be safe to deploy.

### CODE REVIEW

**Comment by axelsson** [comment]
[Update rails app user menu to include CAT](https://trello.com/c/7npoUh4D/948-update-rails-app-user-menu-to-include-cat)

**Comment by axelsson** [comment]
[Update rails app user menu to include CAT](https://trello.com/c/7npoUh4D/948-update-rails-app-user-menu-to-include-cat)

File: app/views/shared/_header.html.erb [hunk 1]
```diff
@@ -24,29 +24,42 @@
...
                       <% end %>
                       <li><%= link_to t('settings'), edit_user_registration_path , class: 'link-ui' %></li>
                       <hr class="hidden d:block d:dropdown-separator">
-                      <% if current_user.organization.present? && current_user.organization.key == 'goclimate' %>
-                        <li><%= link_to t('views.shared.header.navigation.business_dashboard'), dashboard_organization_path(current_user.organization.key) , class: 'link-ui' %></li>
+                      <% if show_business_section %>
+                        <li class="span cursor-default p-2 text-sm text-gray-shade-2"><%= current_user.organization.name %></li>
+                        <% if current_user.organization&.partner? %>
+                          <% if current_user.organization.key == 'euma' %>
+                            <li><%= link_to 'Employee dashboard', "/organization/euma/member-dashboard", class: 'link-ui' %></li>
```
Comment (kallenilver) [inline app/views/shared/_header.html.erb:53]: Could we change `Employee` to `Member` here?

File: public/i18n/en.json [hunk 2]
```diff
@@ -2008,7 +1993,7 @@
             "self_serve": "Self-serve climate reporting"
           },
           "navigation": {
-            "business_dashboard": "Business Dashboard",
+            "cat": "Climate Action Tool",
```
Comment (kallenilver) [inline public/i18n/en.json:1996]: I wonder if people will understand Climate Action Tool in the context of the menu, maybe it's better to say `Business Tool` to start with? But we also want to be consistent.. Maybe deploy as is and we'll see how it feels in prod?

**Review by kallenilver — APPROVED** [review]
A few comments, nothing stopping. Nice work! 🔥

File: public/i18n/en.json (hunk 2 above)
Comment (axelsson) [reply public/i18n/en.json:1996]: I'm thinking that by the time they've logged in with access to CAT they will have either used the current business form to sign up with, where we talk about CAT, or have gotten an email saying how they now have access to CAT so hopefully it already means something to them.

**Pull request merged**